README.md

```diff
@@ -13,7 +13,7 @@
 
 ## What is Yomitan?
 
-Yomitan turns your web browser into a tool for building language literacy by helping you **read** texts that would otherwise be too difficult to tackle in [a variety of supported languages](#multilanguage-support).
+Yomitan turns your web browser into a tool for building language literacy by helping you **read** texts that would otherwise be too difficult to tackle in [a variety of supported languages](./docs/supported-languages.md).
```

Comment:
Our broken link checker cannot catch this? 
Also, a link to docs/supported-languages.md may be good in the Helpful Information section of the README.

Reply:
afaik it only checks for http/https links